src/components/input-field/input-field.tsx

```diff
@@ -235,12 +235,8 @@ export class InputField {
         this.handleCloseMenu = this.handleCloseMenu.bind(this);
         this.setFocus = this.setFocus.bind(this);
 
-        const debounceTimeout = 100;
-        this.changeEmitter = debounce(this.changeEmitter, debounceTimeout, {
-            leading: false,
-            maxWait: 300,
-            trailing: true,
-        });
```

Comment:
I was able to "solve this" (not being able to repro it anymore) by removing the `maxWait` time and changing the debounce timeout to 300.

The default values for `leading` and `trailing` is already set to `false` and `true`, thus the whole config object was removed.

Reply:
Nice!

I'll have my best go at "breaking" it while testing 😝